build/karma.conf.js

```diff
@@ -87,6 +96,9 @@ module.exports = function(config) {
       // Make 'import ...' statements relative to the following paths.
       paths: [conf.paths.frontendSrc, conf.paths.frontendTest],
       transform: [
+        // Browserify transform for the istanbul code coverage tool. Isparta istrumenter for ES6
+        // code coverage.
+        ['browserify-istanbul', {'instrumenter': require('isparta')}],
```

Comment:
I've tried that but couldn't make it work. Don't know why but when I pass imported isparta object here then it doesn't work. This is the only way I was able to make it work.